Example/README.md

```diff
@@ -0,0 +1,37 @@
+# FaunaDB Driver Example Project
+
+This is a simple CRUD project we made in order to show how to use the Swift
+driver along with a iOS project.
+
+## Requirements
+
+- [CocoaPods](https://cocoapods.org/)
+
+## Running the example
+
+- Run `pod install` to setup the project
+- Using XCode 8+, open the workspace `Example.xcworkspace`
+- Edit (or duplicate) the scheme "Example" setting your key's secret at the
+  environment variable "FAUNA_ROOT_KEY"
+
+## Implementation
+
+This demo is a simple blog post manager. It has a basic model called `Post` that
+must have a title and optionally have a body.
```

Comment:
needs a little fixing I think:

...called `Post`, instances of which must have a title and may optionally have a body